output/batch/csv/formatter.go

```diff
@@ -0,0 +1,46 @@
+package csv
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/execution"
+	"github.com/cube2222/octosql/output/batch"
+)
+
+func TableFormatter(separator rune) batch.TableFormatter {
+	return func(w io.Writer, records []*execution.Record, watermark time.Time, errToPrint error) error {
+		if errToPrint != nil {
```

Comment:
Last 1 thing, in table formatter this is printed in the end of the function and here in the beginning, can You change it so they appear in the same place (and Formatters shapes remain similar)?

Reply:
It's intentional. In practice it fits better because of how they get displayed.